docs/data/data_tools.md

```diff
@@ -131,7 +160,7 @@ fsspec integrates directly with scientific Python libraries like **xarray**, **z
 
 ### Installation
 
-fsspec and gcsfs should already be available in the **Base Pangeo Notebook** environment on the JupyterHub.\
+fsspec and gcsfs should already be available in the LEAP-Pangeo Images on the JupyterHub.
```

Comment:
I think we should keep `Base Pangeo Notebook` since that what is displayed in the 2i2c hub selector.